examples/tasks/complex-example.yaml

```diff
@@ -0,0 +1,55 @@
+--- # The Header starts after this line
+  version: 1        # All tasks, like plugins, have versioning
+  schedule:         # How frequently should we run this task?
+    type: "simple"  # Simple means run forever at the interval below
+    interval: "1s"  # This accepts seconds ('s') and milliseconds  ('ms')
+  workflow:
+   collect:
+     metrics:
+       /intel/pcm/EXEC: {}
+       /intel/pcm/FREQ: {}
+       /intel/pcm/INST: {}
+       /intel/pcm/INSTnom: {}
+       /intel/pcm/INSTnom%: {}
+       /intel/psutil/load/load1: {}
+       /intel/psutil/load/load5: {}
+       /intel/psutil/vm/available: {}
+       /intel/psutil/vm/free: {}
+       /intel/psutil/vm/used: {}
+       /intel/haproxy/info/Idle_pct: {}
+       /intel/haproxy/info/MaxConnRate: {}
+       /intel/haproxy/info/MaxSessRate: {}
+       /intel/haproxy/info/MaxSslConns: {}
+       /intel/linux/docker/cpu_stats/cpu_usage/total_usage: {}
+       /intel/linux/docker/cpu_stats/cpu_usage/usage_in_kernelmode: {}
+       /intel/linux/docker/cpu_stats/cpu_usage/usage_in_usermode: {}
+       /intel/linux/docker/cpu_stats/throttling_data/periods: {}
+       /intel/linux/docker/cpu_stats/throttling_data/throttled_periods: {}
+       /intel/linux/docker/cpu_stats/throttling_data/throttled_time: {}
+     config:
+        /intel:
```

Comment:
If this config section stays, there should be some comments about it. I find it odd to show a config that would apply for every `/intel` metric across different plugins that is actually never used by any plugin. What I see that happens is people will just copy the examples verbatim without knowing what everything exactly does, thinking everything is required when it's actually not.